electron/settingsRenderer.js

```diff
@@ -164,6 +173,37 @@ rivets.binders.authref = (el, val) => {
   el.href = "https://inspector.mtgatracker.com/trackerAuth?code=" + val
 }
 
+rivets.binders.recentcardsbinder = (el, cardsObtained) => {
+  var node;
+  var textNode;
+  var currentCard
+  for(var cardID in cardsObtained) {
+    currentCard = mtga.allCards.findCard(cardID)
+    if(currentCard) {
+      textNode = document.createTextNode(cardsObtained[cardID] + "x " + currentCard.attributes.prettyName);
+    } else {
+      textNode = document.createTextNode(cardsObtained[cardID] + "x " + "card-name-not-found");
```

Comment:
```suggestion
      textNode = document.createTextNode(`${cardsObtained[cardID]}x card-name-not-found (${cardID})`);
```

Reply:
(this one is more than a style change; when we don't know what the card is, at least show the grpId)

Reply:
I forgot about the template literals, that looks much more readable than whatever I wrote haha :)